controller/src/main/java/org/jboss/as/controller/persistence/yaml/YamlConfigurationExtension.java

```diff
@@ -498,7 +498,9 @@ public void processOperation(ImmutableManagementResourceRegistration rootRegistr
                             if (map != null && map.containsKey("index")) {
                                 op.get("index").set((Integer) map.get("index"));
                             }
-                            op.get(VALUE).set(processObjectAttribute((ObjectTypeAttributeDefinition) type, map));
+                            if (map != null) {
+                                op.get(VALUE).set(processObjectAttribute((ObjectTypeAttributeDefinition) type, map));
```

Comment:
@ehsavoie Could you please have a look at this fix? All other calls prevent a null pointer deref, except this one. Is it intended to break here?

Reply:
I think the null check used to be before the check from the index

Reply:
@ehsavoie I'm still in doubt, are you fine with this change then?